restock.lic

```diff
@@ -41,6 +41,9 @@ class Restock
     items_to_restock.each do |item|
       item['buy_num'].times do
         buy_item(item['room'], item['name'])
+        if reget(3, 'Seeing that you are too encumbered')
+          bput('get rune from counter', 'You get a')
```

Comment:
❓ Should this be something like `"get #{item['name']} from counter"`?

Reply:
Yes. I did a quick and direct method. I changed to the suggested method.